pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -83,10 +85,14 @@ func newClusterResourceOverride(client clientset.Interface, config io.Reader) (a
 	}, nil
 }
 
-func wrapLimit(limitRange *kapi.LimitRange, resourceName string, obj runtime.Object) error {
-	// limitranger.Limit(limitRange, resourceName, obj)
```

Comment:
I commented out this line to make the code compile.  What was it doing before?  I think it was setting defaults of some kind.  Does the new impl do that?